src/main.rs

```diff
@@ -210,3 +212,25 @@ fn main() {
         Err(e) => error(e.description()),
     }
 }
+
+struct ExportTargetList<'a> {
+    json_file: Option<&'a str>,
+    csv_file: Option<&'a str>,
+}
+
+fn create_exporter(targets: ExportTargetList) -> Option<ExportManager> {
```

Comment:
I have an idea of how to refactor this slightly, but I can do that in a separate PR.

Reply:
Yeah, this was next on my list, after I brought the markdown branch into PR scope

Reply:
See #46